tests/test_transforms/test_feature_selection/test_feature_importance_transform.py

```diff
@@ -316,7 +317,8 @@ def test_mrmr_right_regressors(relevance_table, ts_with_regressors):
         MRMRFeatureSelectionTransform(
             relevance_table=ModelRelevanceTable(), top_k=3, model=RandomForestRegressor(random_state=42)
         ),
-        MRMRFeatureSelectionTransform(relevance_table=StatisticsRelevanceTable(), top_k=3),
+        MRMRFeatureSelectionTransform(relevance_table=StatisticsRelevanceTable(), top_k=3, fast_redundancy=True),
+        MRMRFeatureSelectionTransform(relevance_table=StatisticsRelevanceTable(), top_k=3, fast_redundancy=False),
     ],
 )
 def test_save_load(transform, ts_with_regressors):
```

Comment:
As I can see we still have some tests to check: `test_params_to_tune`, `test_mrmr_right_regressors`, ``